### ISSUE TITLE
Split config doc into 'basic' and 'advanced'

### ISSUE THREAD

**Issue by RollingStar** [beetbox/beets#4533]
I think we can probably take 10% of these and put them into a 'basic' config page, for stuff normal people might want to change. [Terminal encoding](https://beets.readthedocs.io/en/stable/reference/config.html#terminal-encoding) etc could be in an advanced/expert page.

https://beets.readthedocs.io/en/stable/reference/config.html

**Comment by sampsyo** [comment]
Seems like a reasonable idea! One option might be to leave the (exhaustive) reference page with _everything_ intact… and just create a new "getting started" section with descriptions of the most commonly-used options. That could help a lot!

**Comment by RollingStar** [comment]
Yeah we need to keep the existing page in the same spot to preserve old links to it.

My take on the basic config: https://gist.github.com/RollingStar/17655d981906bef117c5a3ffda88f3ab

❌ Add a comment to config and config-basic warning people to keep them in sync (is there a way to automate this?)
❌ new test for text parity between the two (see below)

ideal structure:

Config-basic frontmatter

Config frontmatter (more advanced stuff like config file location changing)

config-all-items

config-basic-items (strictly a subset of config-all-items)

the in python or whatever you can generate config-basic as a collection of [config-basic-frontmatter, config-all-items-basic-subheads]
